docs/_templates/base.html

```diff
@@ -35,3 +35,8 @@
   }, false);
 </script>
 {% endblock %}
+{% block announce %}
```

Comment:
Can you split this to a separate PR and ask Keren for feedback on colors?

Reply:
Done